### ISSUE TITLE
[BUG - 5.0] HTTP Error 500 immediately after install is done

### ISSUE THREAD

**Issue by xionous** [orangehrm/orangehrm#1344]
**Environment details**
OrangeHRM version: 5.0
OrangeHRM source: Docker
Platform: Docker
PHP version: Tried 8.1.6 and 7.4.26
Database and version: MariaDB 10.4 and 10.5
Web server: Apache 2.4.53 and Nginx 1.20.2

If applicable:
Browser: Chrome

**Describe the bug**
Cannot load website immediately after install is done, Error 500

**To Reproduce**
Install Docker container

**Expected behavior**
Load the website

**What do you see instead:**
Error 500

orangehrm.log

```
[2022-06-22T13:07:58.195080+00:00] orangehrm.ERROR: An exception occurred in the driver: SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo for 17227! failed: Name or service not known [] []
[2022-06-22T13:07:58.195243+00:00] orangehrm.ERROR: #0 /var/www/html/src/vendor/doctrine/dbal/src/Connection.php(1768): Doctrine\DBAL\Driver\API\MySQL\ExceptionConverter->convert(Object(Doctrine\DBAL\Driver\PDO\Exception), NULL) #1 /var/www/html/src/vendor/doctrine/dbal/src/Connection.php(1715): Doctrine\DBAL\Connection->handleDriverException(Object(Doctrine\DBAL\Driver\PDO\Exception), NULL) #2 /var/www/html/src/vendor/doctrine/dbal/src/Connection.php(327): Doctrine\DBAL\Connection->convertException(Object(Doctrine\DBAL\Driver\PDO\Exception)) #3 /var/www/html/src/vendor/doctrine/dbal/src/Connection.php(389): Doctrine\DBAL\Connection->connect() #4 /var/www/html/src/vendor/doctrine/dbal/src/Connection.php(351): Doctrine\DBAL\Connection->getDatabasePlatformVersion() #5 /var/www/html/src/vendor/doctrine/dbal/src/Connection.php(276): Doctrine\DBAL\Connection->detectDatabasePlatform() #6 /var/www/html/src/vendor/doctrine/orm/lib/Doctrine/ORM/Query.php(795): Doctrine\DBAL\Connection->getDatabasePlatform() #7 /var/www/html/src/vendor/doctrine/orm/lib/Doctrine/ORM/Query.php(256): Doctrine\ORM\Query->getQueryCacheId() #8 /var/www/html/src/vendor/doctrine/orm/lib/Doctrine/ORM/Query.php(283): Doctrine\ORM\Query->parse() #9 /var/www/html/src/vendor/doctrine/orm/lib/Doctrine/ORM/AbstractQuery.php(1160): Doctrine\ORM\Query->_doExecute() #10 /var/www/html/src/vendor/doctrine/orm/lib/Doctrine/ORM/AbstractQuery.php(1115): Doctrine\ORM\AbstractQuery->executeIgnoreQueryCache(NULL, NULL) #11 /var/www/html/src/vendor/doctrine/orm/lib/Doctrine/ORM/AbstractQuery.php(914): Doctrine\ORM\AbstractQuery->execute(NULL, NULL) #12 /var/www/html/src/plugins/orangehrmCorePlugin/Authorization/Dao/ScreenDao.php(43): Doctrine\ORM\AbstractQuery->getOneOrNullResult() #13 /var/www/html/src/plugins/orangehrmCorePlugin/Service/MenuService.php(168): OrangeHRM\Core\Authorization\Dao\ScreenDao->getScreen('auth', 'login') #14 /var/www/html/src/plugins/orangehrmCorePlugin/Helper/VueControllerHelper.php(205): OrangeHRM\Core\Service\MenuService->getMenuItems('/web/index.php') #15 /var/www/html/src/plugins/orangehrmCorePlugin/Helper/VueControllerHelper.php(123): OrangeHRM\Core\Helper\VueControllerHelper->getMenuItems() #16 /var/www/html/src/plugins/orangehrmCorePlugin/Controller/AbstractVueController.php(154): OrangeHRM\Core\Helper\VueControllerHelper->getContextParams() #17 /var/www/html/src/plugins/orangehrmCorePlugin/Controller/AbstractVueController.php(184): OrangeHRM\Core\Controller\AbstractVueController->render(Object(OrangeHRM\Framework\Http\Request)) #18 /var/www/html/src/plugins/orangehrmAuthenticationPlugin/Controller/LoginController.php(88): OrangeHRM\Core\Controller\AbstractVueController->handle(Object(OrangeHRM\Framework\Http\Request)) #19 /var/www/html/src/vendor/symfony/http-kernel/HttpKernel.php(152): OrangeHRM\Authentication\Controller\LoginController->handle(Object(OrangeHRM\Framework\Http\Request)) #20 /var/www/html/src/vendor/symfony/http-kernel/HttpKernel.php(74): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(OrangeHRM\Framework\Http\Request), 1) #21 /var/www/html/src/lib/framework/Framework.php(160): Symfony\Component\HttpKernel\HttpKernel->handle(Object(OrangeHRM\Framework\Http\Request), 1, true) #22 /var/www/html/web/index.php(42): OrangeHRM\Framework\Framework->handleRequest(Object(OrangeHRM\Framework\Http\Request)) #23 {main} [] []
```

**Comment by RajithaKumara** [comment]
Hi @xionous ,
I just check and found [this](https://stackoverflow.com/questions/30182984/sqlstatehy000-2002-php-network-getaddresses-getaddrinfo-failed-name-or-ser).
Seems PDO can't resolve MySQL server host name.

1. What is the operating system your Docker environment configured?
2. Also is MySQL server in a container or in the cloud?
3. Please provide output of `php -m`
4. You used image from https://hub.docker.com/r/orangehrm/orangehrm and evaluate locally with https://github.com/orangehrm/orangehrm/wiki/How-to-Install-via-Docker-image steps?

**Comment by xionous** [comment]
Hi, 

My docker environment is running on unRAID which runs on slackware.

The MySQL is local MariaDB 

```
root@2281514689a1:/var/www/html# php -m
[PHP Modules]
Core
ctype
curl
date
dom
fileinfo
filter
ftp
gd
hash
iconv
json
ldap
libxml
mbstring
mysqli
mysqlnd
openssl
pcre
PDO
pdo_mysql
pdo_sqlite
Phar
posix
readline
Reflection
session
SimpleXML
sodium
SPL
sqlite3
standard
tokenizer
xml
xmlreader
xmlwriter
Zend OPcache
zip
zlib

[Zend Modules]
Zend OPcache
```

Yes i used that image and followed the guide.

The weird thing is the installer works fine and it connects to and populates the database but the moment the install finishes I get that error

And i have tested and am able to reach the database from within the container:

```
root@2281514689a1:/var/www/html# ping orangehrm-db
PING orangehrm-db (172.20.0.3) 56(84) bytes of data.
64 bytes from orangehrm-db.orangehrm (172.20.0.3): icmp_seq=1 ttl=64 time=0.067 ms
64 bytes from orangehrm-db.orangehrm (172.20.0.3): icmp_seq=2 ttl=64 time=0.062 ms
64 bytes from orangehrm-db.orangehrm (172.20.0.3): icmp_seq=3 ttl=64 time=0.036 ms
64 bytes from orangehrm-db.orangehrm (172.20.0.3): icmp_seq=4 ttl=64 time=0.035 ms
```

**Comment by xionous** [comment]
@RajithaKumara Any thoughts?

**Comment by RajithaKumara** [comment]
Hi @xionous ,
took a little long, but can you check;
https://github.com/orangehrm/orangehrm/issues/1501#issuecomment-1287601899
https://github.com/orangehrm/orangehrm/issues/1501#issuecomment-1287618872

**Comment by RajithaKumara** [comment]
This issue was fixed in OrangeHRM 5.3. Get the latest build [here](https://sourceforge.net/projects/orangehrm/).
